Source/Core/InputCommon/ControllerInterface/SDL/SDL.cpp

```diff
@@ -304,6 +250,37 @@ struct SDLMotionAxis
 using SDLMotionAxisList = std::array<SDLMotionAxis, 6>;
 
 // clang-format off
+static constexpr std::array<const char*, 21> s_sdl_button_names = {
+    "Button S",    // SDL_CONTROLLER_BUTTON_A
+    "Button E",    // SDL_CONTROLLER_BUTTON_B
+    "Button W",    // SDL_CONTROLLER_BUTTON_X
+    "Button N",    // SDL_CONTROLLER_BUTTON_Y
```

Comment:
I don't think calling `SDL_CONTROLLER_BUTTON_A` `B` `X` `Y`, "S" "E" "W" "N" is a good idea. Can we really assume those are the physical positions of those buttons? Don't even Nintendo gamepads differ here? What does SDL do for Sony gamepads?

Reply:
Are you saying a switch pro controller's physical button B will activate "Button S" despite being called `SDL_CONTROLLER_BUTTON_A`?

Reply:
Exactly.